aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SubscriptionProcessor.java

```diff
@@ -56,15 +82,47 @@
      * Start subscribing to model mutations.
      */
     void startSubscriptions() {
-        disposable.add(remoteModelMutations.observe()
+        Set<Observable<SubscriptionEvent<? extends Model>>> subscriptions = new HashSet<>();
+        for (Class<? extends Model> clazz : modelProvider.models()) {
+            for (SubscriptionType subscriptionType : SubscriptionType.values()) {
+                subscriptions.add(subscriptionObservable(appSync, subscriptionType, clazz));
+            }
+        }
+        disposable.add(Observable.merge(subscriptions)
+            .subscribeOn(Schedulers.io())
+            .observeOn(Schedulers.io())
             .subscribe(
-                mutationOnSubscription -> {
-                    buffer.onNext(mutationOnSubscription);
-                    LOG.info("Successfully enqueued mutation from subscription: " + mutationOnSubscription);
-                },
-                error -> LOG.warn("Error enqueuing mutation from subscription.", error),
-                () -> LOG.warn("Subscription to remote model mutations is completed.")
-            )
+                buffer::onNext,
+                buffer::onError,
+                buffer::onComplete
+            ));
+    }
+
+    @SuppressWarnings("unchecked")
+    private static <T extends Model> Observable<SubscriptionEvent<? extends Model>>
+            subscriptionObservable(AppSync appSync, SubscriptionType subscriptionType, Class<T> clazz) {
+        return Observable.<GraphQLResponse<ModelWithMetadata<T>>>create(emitter -> {
+            CountDownLatch latch = new CountDownLatch(1);
+            SubscriptionMethod method = subscriptionMethodFor(appSync, subscriptionType);
+            AtomicReference<Cancelable> cancelable = new AtomicReference<>(NoOpCancelable::new);
+            emitter.setCancellable(cancelable::get);
+            cancelable.set(method.subscribe(
+                clazz,
+                token -> latch.countDown(),
+                emitter::onNext,
+                emitter::onError,
+                emitter::onComplete
+            ));
+            latch.await(SUBSCRIPTION_START_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+        })
```

Comment:
To incorporate, my changes all you would have to do here is:

```
        .doOnError(exception -> {
            LOG.warn("An error occurred with the subscription for model " + clazz.getSimpleName() +
                " and operation type " + subscriptionType.name(), exception.getCause());
        })
        .onErrorResumeNext((ObservableSource<GraphQLResponse<ModelWithMetadata<T>>>) Observer::onComplete)
```
This basically handles the error by logging it and completing the observable. This prevents the Orchestrator from crashing if there's no API configured or the API is not reachable.

If you think it makes sense, we can incorporate this changes here.

That's basically what I added in #459